README.md

```diff
@@ -8,45 +8,49 @@ CityCoins give communities the power to improve and program their cities.
 
 ### Tests
 
-In order to be able to fully unit test the functionality within the mining and stacking contracts
-the tests run in one of two modes;
+To be able to fully unit test the functionality contracts with external references (to deployed mainnet contracts)
+are copied to a test folder - `tests/contracts/extensions` and the external references replaced by internal testable
+target contracts (or stubs). The copy / filter task is automated by running the script here;
 
-- local
-- default
-
-The default way to run the tests is to copy Clarinet.toml.default to Clarinet.toml.
+```bash
+bash scripts/copy-contracts.sh
+```
 
-This will run the tests against the two deployed contracts;
+NB If you are unable to execute the script, run the following command to update the permissions then try again:
 
 ```bash
-contracts/extensions/ccd006-city-mining.clar
-contracts/extensions/ccd007-city-stacking.clar
+chmod 755 scripts/copy-contracts.sh
 ```
 
-By copying Clarinet.toml.local to Clarinet.toml the test will be run against;
+The difference between running the tests with the local, filtered contracts is just the following changes
+in Clarinet.toml;
 
 ```bash
-tests/contracts/extensions/ccd006-city-mining.clar
-tests/contracts/extensions/ccd007-city-stacking.clar
-```
+[contracts.ccd006-city-mining]
+path = "tests/contracts/extensions/ccd006-city-mining.clar"
 
-The two sets of contracts differ only in that references to mainnet contracts in the former have been
-replaced with references to local sip-010 test contracts.
+[contracts.ccd007-city-stacking]
+path = "tests/contracts/extensions/ccd007-city-stacking.clar"
+```
```

Comment:
Noting this part might change pending convo here: https://github.com/citycoins/protocol/pull/42#issuecomment-1372381619